saleor/plugins/manager.py

```diff
@@ -141,10 +143,15 @@ def calculate_checkout_shipping(
             "calculate_checkout_shipping", default_value, checkout, lines, discounts
         )
 
-    def calculate_order_shipping(self, order: "Order") -> TaxedMoney:
+    def calculate_order_shipping(self, order: "Order", channel_id) -> TaxedMoney:
```

Comment:
Maybe we should get channel_id form order instead of passing it as param? 
In which situation we should have different channel id than `order.channel`